frontend/src/router/index.js

```diff
@@ -296,7 +296,7 @@ const router = createRouter({
           name: 'manage-users',
           component: ManageUsersView,
           meta: {
-            permission: PERMISSIONS.MANAGE_USERS,
+            permission: PERMISSIONS.VIEW_USERS,
```

Comment:
I can't find VIEW_USERS permission in permissions.js. should it be MANAGE_USERS_VIEW?

Reply:
Yeah. My bad. Thought this would have failed but I guess it resolves to undefined so it's like having no permission defined on meta. Fixed.